src/link/edit.js

```diff
@@ -193,18 +210,33 @@ const Edit = ( {
 		}
 	}, [ value ] );
 
+	useEffect( () => {
+		const tooltipDisplayedCount = parseInt( wikipediapreviewCustomTooltip.tooltipCount );
```

Comment:
I think it would be better to do the parseInt in one place only.

Reply:
Done: https://github.com/wikimedia/wikipediapreview-wordpress/pull/111/commits/51680afc81cc3460a81091d557e2026a56f50b89#diff-ae84250cfe3c1ac6bf53e94b3d6e7476485fc286f821e8f1ec1faea8f7039cabR10